src/fsharp/service/SemanticClassification.fsi

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.SourceCodeServices
+
+[<RequireQualifiedAccess>]
+type SemanticClassificationType =
+    | ReferenceType
+    | ValueType
+    | UnionCase
+    | Function
+    | Property
+    | MutableVar
+    | Module
+    | Printf
+    | ComputationExpression
+    | IntrinsicFunction
+    | Enumeration
+    | Interface
+    | TypeArgument
+    | Operator
+    | Disposable
+
+[<AutoOpen>]
+module internal TcResolutionsExtensions =
```

Comment:
Would be good to have `///` comments in this file too

Reply:
Yes. I'll put a comment here.